### DISCUSSION TITLE
Episode 162: Sean Parent on the History of Pascal

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#54] [Episode]
Episode Link: https://adspthepodcast.com/2023/12/29/Episode-162.html

Feel free to write comments or ask questions 🙂

**Comment by jdebache** [comment]
I don't think the guy would appreciate a C++ podcast named after his book: https://youtu.be/SUgrS_KbSI8?si=P-9q_ufU0rJRz0N8&t=5781 🤣

**Reply by codereport** [reply]
This is hilarious. Will definitely mention this in a future episode : D
